commons/src/main/kotlin/com/simplemobiletools/commons/helpers/BaseConfig.kt

```diff
@@ -549,4 +549,13 @@ open class BaseConfig(val context: Context) {
     var isCustomOrderSelected: Boolean
         get() = prefs.getBoolean(FAVORITES_CUSTOM_ORDER_SELECTED, false)
         set(selected) = prefs.edit().putBoolean(FAVORITES_CUSTOM_ORDER_SELECTED, selected).apply()
+
+    var viewType: Int
+        get() = prefs.getInt(VIEW_TYPE, VIEW_TYPE_LIST)
+        set(viewType) = prefs.edit().putInt(VIEW_TYPE, viewType).apply()
+
+    var contactsGridColumnCnt: Int
+        get() = prefs.getInt(CONTACTS_GRID_COLUMN_COUNT, CONTACTS_GRID_DEFAULT_COLUMNS_COUNT)
+        set(gridLayoutSpanCount) = prefs.edit().putInt(CONTACTS_GRID_COLUMN_COUNT, gridLayoutSpanCount).apply()
```

Comment:
use the same variable names in configs on line 557 and 559. Either gridLayoutSpanCount or contactsGridColumnCnt

Reply:
Oh, completely forgot about it. Thanks.